doc/rados/index.rst

```diff
@@ -16,7 +16,7 @@ and write data to the Ceph Storage Cluster.
 .. raw:: html
 
 	<style type="text/css">div.body h3{margin:5px 0px 0px 0px;}</style>
-	<table cellpadding="10"><colgroup><col width="33%"><col width="33%"><col width="33%"></colgroup><tbody valign="top"><tr><td><h3>Config and Deploy</h3>
+	<table cellpadding="10"><colgroup><col width="33%"><col width="33%"><col width="33%"></colgroup><tbody valign="top"><tr><td><h3>Configuration</h3>
```

Comment:
I think the better way to dedeup this is to move Cephadm into the deployment section.

Reply:
I see two valid points where we can hook in cephadm (from a docs perspective):

1. At [toplevel](https://docs.ceph.com/en/latest/), like it is right now. Similar to ceph-volume or so. 
2. Hide it in the [deployment](https://docs.ceph.com/en/latest/install/) chapter of the docs.

I think there are valid arguments for putting it in both of them. But I don't know if it is wise to replicate the deployment toctree within [rados](https://docs.ceph.com/en/latest/rados/). That's why I removed it. In my opinion, there is still too much deployment specific docs under rados, like [operating](https://docs.ceph.com/en/latest/rados/operations/operating/#operating-a-cluster)

Reply:
i am for option 2. from a new ceph user's perspective, i don't even know what cephadm or ceph-volume is. what i want is to deploy ceph, deploy some certain services, or replace one or more osds. if cephadm were named "ceph installer", user would be more likely to click cephadm in the toc. that's the reason why think cephadm should be accessible in https://docs.ceph.com/en/latest/rados/.

IMHO, it also difficult to decouple operating from cephadm. as it is a tool for facilitating both the deployment and operating. i think what we need is to consolidate them.

Reply:
> i am for option 2. from a new ceph user's perspective, i don't even know what cephadm or ceph-volume is. what i want is to deploy ceph, deploy some certain services, or replace one or more osds. if cephadm were named "ceph installer", user would be more likely to click cephadm in the toc. that's the reason why think cephadm should be accessible in https://docs.ceph.com/en/latest/rados/.
> 
> IMHO, it also difficult to decouple operating from cephadm. as it is a tool for facilitating both the deployment and operating. i think what we need is to consolidate them.

I have the impression that this is not going to work. The ceph docs need to cope with at least four different types of deployment:

1. cephadm based using (`ceph orch restart mds ...`)
2. traditional systemd based deployment which is often used as an example in rados (`systemctl restart ceph-mds.target`)
3. Rook based (`kubectl get pods -l app=rook-ceph-mds`)
3. BSD to some degree

Note that cephadm not being 100% compatible with the traditional systemd based deployment. We have a conflict here already. I don't think `https://docs.ceph.com/en/latest/rados/` should care about the deployment and instead should try to be agnostic.

Reply:
> should try to be agnostic.

@sebastian-philipp i hear you. "to be agnostic" means "not opinioned", but every deployment approach is opinioned. i just want to ensure that cephadm could be more visible when user is looking for a deployment approach for Ceph.

anyway i don't want to block you.

Reply:
My problem is, I have a hard time making sense out of doc/rados/index.rst`. It's called "*Ceph Storage Cluster*" in the docs., thus we're talking about the Ceph cluster as an entity in this chapter. Remember we're making a first impression on users here. This is a page for beginners. 

* But why are only the librados APIs part of this chapter? 
* The introduction sentence 
   > Ceph Storage Clusters consist of two types of daemons: a Ceph OSD Daemon (OSD) stores data as objects on a storage node; and a Ceph Monitor (MON)"

  is just wrong.  We have so many different kinds of daemons. Cehpadm can deploy maybe 10 different types of daemons. 
* The contents of the columns is questionable for beginners. For example we're linking to the [Filestore config reference](https://docs.ceph.com/en/latest/rados/configuration/filestore-config-ref/) which **no one should touch** in production clusters. And Filestore shoudn't be used anyway nowadays. A lot of the topics linked here are super advanced and are only suitable for the real experts. 
* Except for manual deployments, [msg v2](https://docs.ceph.com/en/latest/rados/configuration/msgr2/) should be an implementation detail and should be taken care of by the deployment tool. 

This doesn't make sense to me at all.

Reply:
@sebastian-philipp I say let's merge this change, and then address your concerns about the accuracy of this section's content in subsequent PRs. What do you think?